sdk/core/core-http/src/createSpanLegacy.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// NOTE: we've moved this code into core-tracing but these functions
```

Comment:
@xirzec - after talking with @joheredi I realized I can't actually remove this function. I don't know if I can mark it as deprecated in some official way (I saw there's a jsdoc for 'deprecated') but I can't outright remove it since most code calls createSpan unconditionally (ie, even if they're not using tracing). So it has to exist, at the very least. I could stub it out so it does nothing but for now it just forwards the call onto core-tracing instead.